src/connectors/inventory/impl.js

```diff
@@ -90,6 +91,57 @@ module.exports = new class extends Connector {
         return validate(transformed);
     }
 
+    async getSystemProfileBatch (ids = [], refresh = false, retries = 2) {
+        if (ids.length === 0) {
+            return {};
+        }
+
+        ids = _.sortBy(ids);
+
+        if (ids.length > pageSize) {
+            const chunks = _.chunk(ids, pageSize);
+            const results = await P.map(chunks, chunk => this.getSystemProfileBatch(chunk, refresh));
+            return _.assign({}, ...results);
+        }
+
+        const uri = this.buildHostsUri();
+        uri.segment(ids.join());
+        uri.segment('system_profile');
+        uri.addQuery('per_page', String(pageSize));
+
+        let response = null;
+
+        try {
+            response = await this.doHttp({
+                uri: uri.toString(),
+                method: 'GET',
+                json: true,
+                rejectUnauthorized: !insecure,
+                headers: this.getForwardedHeaders()
+            },
+            {
+                key: `remediations|http-cache|inventory|${ids.join()}`,
```

Comment:
this is the same key as in getSystemDetailsBatch above i.e. these two functions will keep overwriting the cache entry